staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -483,11 +468,13 @@ See http://k8s.io/docs/reference/using-api/api-concepts/#conflicts`, err)
 
 			if !o.DryRun {
 				// Then create the resource and skip the three-way merge
-				options := metav1.CreateOptions{}
+				helper := resource.NewHelper(info.Client, info.Mapping)
 				if o.ServerDryRun {
-					options.DryRun = []string{metav1.DryRunAll}
+					if err := resource.VerifyDryRun(info.Mapping.GroupVersionKind, o.DynamicClient, o.DiscoveryClient); err != nil {
+						helper.DryRun(o.ServerDryRun)
+					}
 				}
-				obj, err := resource.NewHelper(info.Client, info.Mapping).Create(info.Namespace, true, info.Object, &options)
+				obj, err := helper.Create(info.Namespace, true, info.Object, nil)
```

Comment:
I'm curious if anything is using the option here at all in `helper.Create`. If not, it might be good to just plain remove it.

Reply:
Removed! Only diff used the option, so added CreateWithOptions